x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/compare_fields_table/helpers.test.tsx

```diff
@@ -117,31 +116,6 @@ describe('helpers', () => {
           expect(screen.queryByTestId('typePlaceholder')).not.toBeInTheDocument();
         });
       });
-
-      describe('when `type` is undefined', () => {
-        beforeEach(() => {
-          const withUndefinedType = {
-            ...eventCategory,
-            type: undefined, // <--
-          };
-          const columns = getEcsCompliantTableColumns();
-          const typeRender = columns[1].render;
-
-          render(
-            <TestProviders>
-              <>{typeRender != null && typeRender(withUndefinedType.type, withUndefinedType)}</>
-            </TestProviders>
-          );
-        });
-
-        test('it does NOT render the `type`', () => {
-          expect(screen.queryByTestId('type')).not.toBeInTheDocument();
-        });
-
-        test('it renders the placeholder', () => {
-          expect(screen.getByTestId('typePlaceholder')).toHaveTextContent(EMPTY_PLACEHOLDER);
-        });
-      });
```

Comment:
For reviewers:

This test was removed because it tests an impossible state. Details:
https://github.com/elastic/kibana/pull/185025/files#r1631423077